pkg/plugin/interface.go

```diff
@@ -62,16 +61,6 @@ type Interface interface {
 	SkipCleanup() bool
 }
 
-// Definition defines a plugin's features, method of launch, and other
-// metadata about it.
-type Definition struct {
```

Comment:
This is the main thing I just removed here; it is just a flattened version of the `manifest.Manifest` except it doesnt have the `Driver` field. It leads to us having an extra layer to transfer data through and doesn't provide any clear benefit.